.github/workflows/security-scan.yaml

```diff
@@ -32,20 +32,24 @@ jobs:
       - run: sleep 8
 
       - name: Run ZAP Full Scan on the accounts website
+        id: zap_scan
         # https://www.zaproxy.org/docs/docker/full-scan/
-        run: docker run
-          --add-host host.docker.internal:host-gateway
-          -v $(pwd):/zap/wrk/:rw
-          -u root
-          -t
-          ghcr.io/zaproxy/zaproxy:stable
-          zap-full-scan.py
-          -t http://host.docker.internal:3000
-          -j
-          -g gen.conf
-          -r accounts.zap-scan-report.html
-          -a || true
-          # || true - force exit code 0 so action continues
+        run: |
+          set +e        
+          docker run \
+          --add-host host.docker.internal:host-gateway \
+          -v $(pwd):/zap/wrk/:rw \
+          -u root \
+          -t \
+          ghcr.io/zaproxy/zaproxy:stable \
+          zap-full-scan.py \
+          -t http://host.docker.internal:3000 \
+          -j \
+          -g gen.conf \
+          -r accounts.zap-scan-report.html \
+          -a
+        EXIT_CODE="$?"
+        echo "EXIT_CODE=$EXIT_CODE" >> $GITHUB_OUTPUT
 
       - name: Run Dastardly scan on accounts website
         # https://portswigger.net/burp/documentation/dastardly/generic
```

Comment:
We also need to get thee exit code from the 2x Dastardly scans if they provide this information.